numpy/core/src/multiarray/textreading/tokenize.h

```diff
@@ -2,6 +2,10 @@
 #ifndef NUMPY_CORE_SRC_MULTIARRAY_TEXTREADING_TOKENIZE_H_
 #define NUMPY_CORE_SRC_MULTIARRAY_TEXTREADING_TOKENIZE_H_
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
```

Comment:
same include remark here.

Reply:
Thanks moved all the includes out of the guard now.